shoop/themes/classic_gray/plugins.py

```diff
@@ -46,3 +46,32 @@ def get_context_data(self, context):
             "title": self.get_translated_value("title"),
             "products": products
         }
+
+
+class ProductCrossSellsPlugin(TemplatedPlugin):
+    identifier = "classic_gray.product_cross_sells"
+    name = _("Product Cross Sells")
+    template_name = "classic_gray/cross_sells_plugin.jinja"
+    required_context_variables = ["product"]
+    fields = [
+        ("title", TranslatableField(label=_("Title"), required=False, initial="")),
+        ("type", forms.ChoiceField(label=_("Type"), choices=[
+            ("related", "Related"),
+            ("recommended", "Recommended"),
+            ("computed", "Computed"),
+        ], initial="newest")),
+        ("count", forms.IntegerField(label=_("Count"), min_value=1, initial=4)),
```

Comment:
We can, we would just need to add that to the get_product_cross_sells template helper, and so we'd probably want to replace list_visible with _get_list_products, similar to what we did with get_best_selling_products, etc.

Looking at what's currently in get_product_cross_sells, there shouldn't be an issues I don't think